prow_config.yaml

```diff
@@ -23,3 +92,62 @@ workflows:
       - components/crud-web-apps/common/frontend/kubeflow-common-lib/*
```

Comment:
lets also include the directory of the common code here as well
components/crud-web-apps/common/backend